packages/autocomplete-js/src/getDefaultOptions.ts

```diff
@@ -48,32 +57,42 @@ export function getDefaultOptions<TItem extends BaseItem>(
     getListProps,
     getPanelProps,
     getRootProps,
+    panelContainer,
+    panelPlacement,
+    render,
+    renderer,
+    touchMediaQuery,
     ...core
   } = options;
-  const renderer = {
-    container: getHTMLElement(container),
-    panelContainer: panelContainer
-      ? getHTMLElement(panelContainer)
-      : document.body,
-    render: render ?? defaultRenderer,
-    panelPlacement: panelPlacement ?? 'input-wrapper-width',
-    classNames: mergeClassNames(
-      defaultClassNames,
-      classNames ?? {}
-    ) as AutocompleteClassNames,
-    touchMediaQuery: touchMediaQuery ?? '(hover: none) and (pointer: coarse)',
-    getEnvironmentProps: getEnvironmentProps ?? (({ props }) => props),
-    getFormProps: getFormProps ?? (({ props }) => props),
-    getInputProps: getInputProps ?? (({ props }) => props),
-    getItemProps: getItemProps ?? (({ props }) => props),
-    getLabelProps: getLabelProps ?? (({ props }) => props),
-    getListProps: getListProps ?? (({ props }) => props),
-    getPanelProps: getPanelProps ?? (({ props }) => props),
-    getRootProps: getRootProps ?? (({ props }) => props),
+
+  const defaultRenderer: AutocompleteRenderer = {
```

Comment:
should defaultRender & defaultRenderer be created at the same level instead? I suggest both at the root, where defaultRender is

Reply:
Yes! 10489ea580047e10b03a2746ef9fcf5c28da3b77